src/core/src/core_logic/PatchInstaller.py

```diff
@@ -192,7 +228,7 @@ def install_updates(self, maintenance_window, package_manager, simulate=False):
                 break
```

Comment:
This also was a part of the initial steps in install_updates(), happening before actual package install